packages/react-utils/jest.config.js

```diff
@@ -0,0 +1,8 @@
+const baseConfig = require('../../jest.config.base');
+
+module.exports = {
+    ...baseConfig,
+    collectCoverage: true,
+    collectCoverageFrom: ['src/**/*.ts'],
+    testEnvironment: '../../JestCustomEnv.js',
+};
```

Comment:
`react-utils` did not have unit tests at all. I set it up for the first test. Code coverage is not great :smile: